src/test_l3_dsHdmiIn.c

```diff
@@ -0,0 +1,1396 @@
+    {
+         
+         UT_LOG_INFO("Received statuschange callback port:%d , isPortConnected: %s",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, i),
+                 UT_Control_GetMapString(bool_mapTable, inputStatus.isPortConnected[i]));
+    }
+
+    gStatusChange = inputStatus;
+}
+
+/**
+ * @brief Callback function for HdmiIn Video Mode change.
+ *
+ * This function is invoked whenever a Video Mode change occurs in the Hdmi connection.
+ */
+static void hdmiInVideoModeUpdateCB(dsHdmiInPort_t port, dsVideoPortResolution_t videoResolution)
+{
+    UT_LOG_INFO("Received videomode change callback port:%s resolution name: %s, pixelResolution: %s\t" 
+                 "aspectRatio:%s, stereoScopicMode:%s, frameRate:%s, interlaced:%s\t",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port),
+                 videoResolution.name, 
+                 UT_Control_GetMapString(dsVideoResolution_mapTable, videoResolution.pixelResolution),
+                 UT_Control_GetMapString(dsVideoAspectRatio_mapTable, videoResolution.aspectRatio),
+                 UT_Control_GetMapString(dsVideoStereoScopicMode_mapTable, videoResolution.stereoScopicMode),
+                 UT_Control_GetMapString(dsVideoFrameRate_mapTable, videoResolution.frameRate),
+                 UT_Control_GetMapString(bool_mapTable, videoResolution.interlaced));
+
+                 gVideoresolution = videoResolution;
+}
+
+/**
+ * @brief Callback function for HdmiIn Allmmode change.
+ *
+ * This function is invoked whenever a Allmmode change occurs in the Hdmi connection.
+ */
+static void hdmiInAllmChangeCB(dsHdmiInPort_t port, bool allm_mode )
+{
+    UT_LOG_INFO("Received AllmChange status callback port: %s, sigstatus: %s",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port),
+                 UT_Control_GetMapString(bool_mapTable, allm_mode));
+
+    gAllmmode = allm_mode;
+}
+
+/**
+ * @brief Callback function for HdmiIn Allmmode change.
+ *
+ * This function is invoked whenever a Allmmode change occurs in the Hdmi connection.
+ */
+static void hdmiInAVLatencyChangeCB(int audio_latency, int video_latency)
+{
+    UT_LOG_INFO("Received AVlatencyChange status callback audio_latency: %d\t"
+                 "video_latency:%d \t", audio_latency , video_latency);
+
+    gaudio_latency = audio_latency;
+    gvideo_latency = video_latency;
+}
+
+/**
+ * @brief Callback function for HdmiIn Allmmode change.
+ *
+ * This function is invoked whenever a Allmmode change occurs in the Hdmi connection.
+ */
+static void hdmiInAviContentTypeChangeCB(dsHdmiInPort_t port, dsAviContentType_t avi_content_type)
+{
+    UT_LOG_INFO("Received AviContentType change callback port: %s, avi_content_type: %d\t"
+                 "video_latency:%d\t",
+                 UT_Control_GetMapString(dsHdmiInPort_mapTable, port),
+                 UT_Control_GetMapString(dsAviContentType_mapTable, avi_content_type)),
+
```

Comment:
1. Here, in Log print statement you have updated with 3 variables(%s, %d, %d). But only 2 arguments are provided here.

2. And, there is type mismatch Ex:  **avi_content_type: %d** is expecting to be an integer. But here string parameter(UT_Control_GetMapString) has been provided.

3.  After Line No: 362, "comma" is expecting to provide third argument but it is missing. And, this statement is not completed with semicolon(;)

Reply:
done